bin/ingest-gisaid

```diff
@@ -76,16 +76,19 @@ main() {
         aws s3 cp --no-progress "$S3_DST/gisaid.ndjson.gz" - | gunzip -cfq > data/gisaid.ndjson
     fi
 
+    flagged_annotations="$(mktemp -t flagged-annotations-XXXXXX)"
+    trap "rm -f '$flagged_annotations'" EXIT
     ./bin/transform-gisaid data/gisaid.ndjson \
-      --output-metadata data/gisaid/metadata.tsv \
-      --output-fasta data/gisaid/sequences.fasta
+        --output-metadata data/gisaid/metadata.tsv \
+        --output-fasta data/gisaid/sequences.fasta > "$flagged_annotations"
 
     ./bin/flag-metadata data/gisaid/metadata.tsv > data/gisaid/flagged_metadata.txt
     ./bin/check-locations data/gisaid/metadata.tsv \
         data/gisaid/location_hierarchy.tsv \
         gisaid_epi_isl
 
     if [[ "$branch" == master ]]; then
+        ./bin/notify-slack --upload "flagged-annotations" < "$flagged_annotations"
```

Comment:
The printed warnings in Tony's `transform-gisaid` are now sent to Slack, but only when on the master branch. I just wanted to check that this is the desirable behavior. @eharkins

Reply:
This makes sense to me!